src/CodeStyle/Core/CodeFixes/FormattingCodeFixHelper.cs

```diff
@@ -12,7 +12,7 @@ internal static class FormattingCodeFixHelper
     {
         internal static async Task<Document> FixOneAsync(Document document, OptionSet options, Diagnostic diagnostic, CancellationToken cancellationToken)
         {
-            var tree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            _ = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
I know that would not be safe, but it would be nice to offer it as an option in case the user has determined that it is safe or the code is just useless.

Reply:
It might be better to offer it a refactoring to remove `_ = ...` assignment statement - you probably don't want a FixAll as you would want to make a conscious decision on each redundant statement case.

Reply:
Yeah I imagine the code fix would not have a fix all.